uber/config.py

```diff
@@ -106,18 +106,21 @@ def get_attendee_price(self, dt=None):
         price = self.INITIAL_ATTENDEE
         if self.PRICE_BUMPS_ENABLED:
 
+            for day, bumped_price in sorted(self.PRICE_BUMPS.items()):
+                if (dt or sa.localized_now()) >= day:
```

Comment:
Yeah I changed that on purpose. :) It should be localized time, we were (effectively) always doing that. Just now that the code has moved, we would've switched to UTC.